dockertest/images_unittests.py

```diff
@@ -341,6 +341,15 @@ def test_addr_no_user(self):
         self.assertEqual(user, None)
         self.assertEqual(addr, "address:1234")
 
+    def test_empty_tag(self):
+        test = "myname:"   # note the trailing colon with no actual tag name
```

Comment:
Oh man.  There are about a billion places that assume passing no tag name implies a ``latest`` will show up in the images list.  I predict the adockerclypse is nye.  It may be worthwhile to investigate and see if there are some bugs to file on this.  For one, it breaks backward compatibility with any tooling that relies on the old behavior.  I'm guessing it may also cause ambiguities WRT specifying an empty tag vs no tag.